src/gui/editors/PianoRoll.cpp

```diff
@@ -1772,6 +1773,15 @@ void PianoRoll::mousePressEvent(QMouseEvent * me )
 					// then resize the note
 					m_action = ActionResizeNote;
 
+					m_minResizeLen = quantization();
+					for (Note *note : getSelectedNotes())
+					{
+						int thisMin = note->oldLength() % quantization();
```

Comment:
```suggestion
						if (note->oldLength() <= 0) { note->setOldLength(1); }
						int thisMin = note->oldLength() % quantization();
```
Integrates #5515. Since we already iterate through the selection here we may as well fix this length instead of checking on every drag event.